CONTRIBUTING.md

```diff
@@ -64,12 +64,16 @@ Our commit style is defined in a __[commit template]__. Use it as a reference or
 ### Rules and Recommendations
 
 - *Use your __real name__ and __real email__.* We do not accept anonymous contributions!
+  - Every commit should have author's full legal name (in latin letters, diacritics allowed) with real e-mail address.
+  - Same applies to the author's GitHub profile:
+  1. It should have full name set (in the [Name field here](https://github.com/settings/profile)) that matches one specified in the commits.
+  1. In the [e-mail settings](https://github.com/settings/emails) the checkbox "Keep my email addresses private" must be unchecked.
```

Comment:
Then, contributing to ReactOS would force to disclose all addresses declared on GitHub? Yuck!

Reply:
There is always a possibility to remove addresses from GitHub that you don't want to disclose. Personally I have only two emails associated with GitHub, one of which I use for signing code changes, and another for translations.

Reply:
We obviously don't care at all about your own private email; However, how would one contact you without knowing *some* email? You use your email to contact privately some ReactOS devs there, so the reverse should be possible. Use/create an email address that is meant to be public and tell github to use that latter one as the "public" one (while keeping the one used to manage your own github settings private)...

Reply:
![image](https://user-images.githubusercontent.com/1969829/201959117-1e9b2ba2-5612-49b7-9960-6bf9a97b7a97.png)

![image](https://user-images.githubusercontent.com/1969829/201959129-9dd5079d-a580-493d-b391-041f2cba09ca.png)

Reply:
What @HBelusca said. It's fine to set up one public e-mail address to work on the ReactOS repo, and this should be a requirement.
But disclosing all addresses connected to a GitHub account serves no additional purpose.

Reply:
> Use/create an email address that is meant to be public and tell github to use that latter one as the "public" one (while keeping the one used to manage your own github settings private)...

My Git commits (except the ones I created through GitHub web interface on a couple recent PRs...) do have a usable address.

Your images show a setup where all addresses, including the primary one, are public, which is not the privacy I want to use.

> Also for unknown reasons, this doesn't seem to cause problems with Wine developers, they manage to retain their private emails private, but are able to use public ones.

Interesting. It would be worth asking them how to do that...

Reply:
> set up one public e-mail address to work on the ReactOS repo,

As far as I remember, noone explained how to make that work on GitHub.
See https://github.com/reactos/reactos/pull/4881#discussion_r1295884940, which confirms what I wrote all along.

> disclosing all addresses connected to a GitHub account serves no additional purpose.

This is why I consider new ROS rules to be an infringement and I will not comply with them.